test-prow-e2e.sh

```diff
@@ -32,9 +32,14 @@ oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-h
 DBUS_SESSION_BUS_ADDRESS=/dev/null
 export DBUS_SESSION_BUS_ADDRESS
 
-CHROME_VERSION=$(google-chrome --version) ./test-gui.sh "${1:-e2e}"
+if [ $# -eq 0 ] || [ "$1" != "login" ]; then
+  CHROME_VERSION=$(google-chrome --version) ./test-gui.sh "${1:-e2e}"
+fi
```

Comment:
```suggestion
SCENARIO="${1:=e2e}"
# Only run Cypress tests for the login scenario.
if [ "$SCENARIO" != "login" ]; then
  CHROME_VERSION=$(google-chrome --version) ./test-gui.sh "$SCENARIO"
fi
```

Then you can use `$SCENARIO` again below instead of `[ $# -eq 0 ] || [ "$1" == "release" ]`, etc.

Reply:
thanks, fixed